theano/tensor/basic.py

```diff
@@ -3963,16 +3963,78 @@ def shape_padright(t, n_ones=1):
 
 
 @constructor
-def stack(*tensors):
+def shape_padaxis(t, axis):
+    """Reshape `t` by adding 1 at the dimension `axis`.
```

Comment:
This also need to be documented in the file doc/library/tensor/basic.txt after shape_padright